### ISSUE TITLE
release with tag but no manifest breaks version calculation

### ISSUE THREAD

**Issue by titanous** [flynn/flynn#1803]
If a release fails after tagging but before the manifest is pushed, a subsequent release on the same day will result in this error:

```text
===> 18:55:50.261 building flynn
fatal: tag 'v20150901.0' already exists
```

Instead of using the manifest, we should use the git tags to figure out the iteration number.

**Comment by lmars** [comment]
Duplicate of #1321 (although that issue talks about pushing the tag before starting the release)
